Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -397,7 +397,7 @@ static void generate_to_string(SourceGenerator& scoped_generator, ParameterType
 
         scoped_generator.append(R"~~~(
     if (!@js_name@@js_suffix@.is_undefined()) {
-        if (!@legacy_null_to_empty_string@ || !@js_name@@js_suffix@.is_null())
+        if (!@js_name@@js_suffix@.is_null())
```

Comment:
I don't think it's safe to remove the `legacy_null_to_empty_string` check here, since optional parameters can have a `[LegacyNullToEmptyString]` attribute - see the `features` argument of `window.open()`, for example. I think instead, you shouldn't be including the check if the parameter is nullable.

Reply:
Could you please check it's fine now?